setup.py

```diff
@@ -14,14 +14,14 @@ def get_packages(package):
 
 setup(
     name='zc_events',
-    version='0.3.7-rc4',
+    version='0.3.7-rc5',
     description="Shared code for ZeroCater microservices events",
     long_description='',
     keywords='zerocater python util',
     author='ZeroCater',
     author_email='tech@zerocater.com',
     url='https://github.com/ZeroCater/zc_events',
-    download_url='https://github.com/ZeroCater/zc_events/tarball/0.3.7-rc4',
+    download_url='https://github.com/ZeroCater/zc_events/tarball/0.3.7-rc5',
```

Comment:
Both this and the old `download_url` give me 404s. Are these links valid?

Reply:
yes, but they get made upon release.